pyuoi/linear_model/logistic.py

```diff
@@ -200,6 +202,16 @@ def _pre_fit(self, X, y):
             self.output_dim = 1
         return X, y
 
+    def fit(self, X, y, verbose=False):
```

Comment:
Logistic should already support using non-integer classes. Is this fixing something else?

Reply:
I believe this line chokes on string labels:

https://github.com/BouchardLab/PyUoI/blob/eaf35ffb0a575c95895b37857b0fb66ea2974d2c/pyuoi/linear_model/logistic.py#L255

Reply:
That's true, but that object shouldn't get used on the raw labels (at least in the current implementation). They should already be encoded by the parent UoI_L1Logistic model.
https://github.com/BouchardLab/PyUoI/blob/eaf35ffb0a575c95895b37857b0fb66ea2974d2c/pyuoi/linear_model/logistic.py#L193

Is UoI_L1Logistic broken for you in on the master branch?

Reply:
That line is only two months old. it's possible that master had not yet had that change. I'll get rid  of this change.